modules/serverless-logic-profile-persistence-requirements.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-persistence
+
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-profile-persistence-requirements_{context}"]
+= Profile specific persistence requirements
+
+The persistence configurations provided for both `SonataFlowPlatform` CR and `SonataFlow` CR apply equally to the `preview` and `gitops` profiles. However, these configurations should not be used with the `dev` profile, as this profile ignores them entirely.
+
+The primary difference between the `preview` and `gitops` profiles lies in the build process:
```

Comment:
I think since you're not introducing a format element like a list here, you could just end this sentence with a period. up to you:
```suggestion
The primary difference between the `preview` and `gitops` profiles lies in the build process.
```